constantinCurtis/src/main/java/data/collection/task/classes/AbstractUser.java

```diff
@@ -0,0 +1,70 @@
+package data.collection.task.classes;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+public abstract class AbstractUser {
+    private final String id;
+    private final String username;
+    private Map<String, Address> addresses;
```

Comment:
Please change to protected in order to proper inherit those variables.

Reply:
Please read my teams message